src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -93,6 +95,54 @@ public void given_initiate_tasks_with_follow_up_overdue_category_then_cancel_tas
         taskToTearDown = taskCreatedAsResultOfTheMultipleDmnRule;
     }
 
```

Comment:
if delayTimer is set to future and there is no follow up overdue event, what should happen? Do we need a test for this?